content/blog/2017-11-11-rperseus.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Exploratory Data Analysis of Ancient Texts with rperseus"
+categories: blog
+date: '2017-11-17'
+package_version: 0.0.1
+slug: rperseus
+tags:
+- r
+- API
+- packages
+- textmining
+- text-analysis
+- linguistics
+- data-extraction
+- community
+- software
+- review
+- onboarding
+authors:
+- name: David Ranzolin
+  url: https://github.com/daranzolin
+topicid: 1000
+---
+
+### Introduction
+
+When I was in grad school at Emory, I had a favorite desk in the library. The desk wasn’t particularly cozy or quiet, but what it lacked in comfort it made up for in real estate. My books and I needed room to operate. Students of the ancient world require many tools, and when jumping between commentaries, lexicons, and interlinears, additional clutter is additional “friction”, i.e., lapses in thought due to frustration. Technical solutions to this clutter exist, but the best ones are proprietary and expensive. Furthermore, they are somewhat inflexible, and you may have to shoehorn your thoughts into their framework. More friction.
+
+Interfacing with [the Perseus Digital Library](http://www.perseus.tufts.edu/hopper/) was a popular online alternative. The library includes a catalog of classical texts, a Greek and Latin lexicon, and a word study tool for appearances and references in other literature. If the university library’s reference copy of BDAG[^1] was unavailable, Perseus was our next best thing.
```

Comment:
I can't figure out why the reference numbers aren't showing as superscript since you seem to have used same format as in this post https://ropensci.org/blog/2017/10/06/sholler-plan/.

One suggestion that won't change superscript but good for organization, is to move your references and footnotes to bottom under 
`<!-- references -->`
[^1]: Bauer, Walter. ...
[^2]: The Greek text...
etc

Reply:
Thanks, that is much easier to organize! As for the superscripts, I don't know what's happening either.